src/main/java/vazkii/quark/content/tweaks/module/SimpleHarvestModule.java

```diff
@@ -136,12 +135,13 @@ private BlockState fromString(String key) {
 		}
 	}
 
-	private static void replant(Level world, BlockPos pos, BlockState inWorld, Player player) {
+	private static void harvestAndReplant(Level world, BlockPos pos, BlockState inWorld, Player player) {
+		if (!(world instanceof ServerLevel))
+			return;
+
 		ItemStack mainHand = player.getMainHandItem();
-		boolean isHoe = !mainHand.isEmpty() && mainHand.getItem() instanceof HoeItem;
 
-		BlockState newBlock = crops.get(inWorld);
-		int fortune = HoeHarvestingModule.canFortuneApply(Enchantments.BLOCK_FORTUNE, mainHand) && isHoe ?
+		int fortune = HoeHarvestingModule.canFortuneApply(Enchantments.BLOCK_FORTUNE, mainHand) ?
```

Comment:
note: the `isHoe` check has been removed from this terniary as the check itself is done within `HoeHarvestingModule.canFortuneApply`